semver/_shared.ts

```diff
@@ -127,26 +127,16 @@ const XRANGE_IDENTIFIER = `${NUMERIC_IDENTIFIER}|x|X|\\*`;
 export const XRANGE_PLAIN =
   `[v=\\s]*(?<major>${XRANGE_IDENTIFIER})(?:\\.(?<minor>${XRANGE_IDENTIFIER})(?:\\.(?<patch>${XRANGE_IDENTIFIER})(?:${PRERELEASE})?${BUILD}?)?)?`;
 
-export const XRANGE_REGEXP = new RegExp(
-  `^(?<operator>${COMPARATOR})\\s*${XRANGE_PLAIN}$`,
+// Meaning is "reasonably at or greater than" or "at least and backwards compatible with"
+export const OPERATOR_REGEXP = new RegExp(
```

Comment:
@timreichen Is this change of variable name intentional? (`XRANGE_REGEXP` -> `OPERATOR_REGEXP`)

Reply:
Yes, it is specifically an xrange with operators and needs to be from `XRANGE` without the operator. Would you prefer `OPERATOR_XRANGE_REGEXP`?

Reply:
Fair enough.

> Would you prefer OPERATOR_XRANGE_REGEXP?

That sounds clearer to me!

Reply:
https://github.com/denoland/deno_std/pull/4056